### PR TITLE
fix(eslint): disable prevent abbreviations

### PR DESCRIPTION
**Pull request by stevensacks**
Preventing abbreviations is incompatible with React standards (props, ref, etc.). Additionally, there are many scenarios where abbreviations are acceptable.

In the future, we may decide to make a whitelist, but since that will take time to decide and this rule is currently in the published ruleset, we are releasing it with the rule disabled for now to address this problem quickly.

### CODE REVIEW

**Comment by JAdshead** [comment]
This should probably be of type `feat` unless preventing abbreviations is really breaking

**Comment by stevensacks** [comment]
> This should probably be of type `feat` unless preventing abbreviations is really breaking

Technically, it is. You get an error when you useRef or <Component ref={}/>

File: index.js [hunk 1]
```diff
@@ -116,6 +116,11 @@ module.exports = {
     // regular expressions.
     'unicorn/no-unsafe-regex': 'error',
 
+    // Preventing abbreviations is incompatible with React standards (props, ref, etc.)
+    // Additionally, there are many scenarios where abbreviations are acceptable
```
Comment (PixnBits) [inline index.js:120]: maybe
```suggestion
    // Additionally, there are many popular libraries that have abbreviations as their standard practice
    // this rule prevents that consistency
```

Comment (stevensacks) [reply index.js]: Done

**Comment by JamesSingleton** [comment]
@stevensacks you just need to resolve the conflicts here.

**Comment by stevensacks** [comment]
@PixnBits I made the change you requested. Conflicts resolved.

**Pull request merged**